apps/bixarena/api/src/main/java/org/sagebionetworks/bixarena/api/service/LeaderboardService.java

```diff
@@ -147,13 +149,15 @@ private Sort createSort(LeaderboardSearchQueryDto query) {
     // Map API sort fields to entity fields
     String entityField =
       switch (sortField) {
-        case "bt_score" -> "btScore";
-        case "vote_count" -> "voteCount";
         case "created_at" -> "createdAt";
-        case "model_name" -> "model.name";
         default -> "rank";
       };
 
+    if ("rank".equals(sortField)) {
+      // Create primary rank sort, and add secondary btScore sort
+      return Sort.by(direction, entityField).and(Sort.by(Sort.Direction.DESC, "btScore"));
```

Comment:
Since the rank is inversely proportional to btScore, we should likely use the specified `direction` for sorting btScore too so that we perfectly reverse the order when switching from one direction to the other one.

- If sort direction for rank is ASC, use DESC to sort btScore
- If sort direction for rank is DESC, use ASC to sort btScore